qa/standalone/mon/mon-cluster-log.sh

```diff
@@ -145,9 +145,17 @@ function TEST_journald_cluster_log_level() {
     ceph osd down 0
     TIMEOUT=20 wait_for_osd up 0 || return 1
     search_str="osd.0.*boot"
-    sudo journalctl _COMM=ceph-mon CEPH_CHANNEL=cluster PRIORITY=6 --output=json-pretty --since "60 seconds ago" |jq '.MESSAGE' > $dir/journal.log
-    grep -q "$search_str" $dir/journal.log
-    return_code=$?
+    return_code=1
+    TIMEOUT=60 
+    for ((i=0; i < $TIMEOUT; i++)); do
+        sudo journalctl _COMM=ceph-mon CEPH_CHANNEL=cluster PRIORITY=6 --output=json-pretty --since "60 seconds ago" |jq '.MESSAGE' > $dir/journal.log
+        if ! grep "$search_str" $dir/journal.log; then
+            sleep 1
+        else
+            return_code=0
+            break
+        fi
+    done
```

Comment:
I didn't find any existing helper function which pipes the logs to the file as well as greps for the desired string. Please let me know if I should instead create a helper func for the above instead and use it here.

Reply:
@shraddhaag indent - 2 spaces
except for that test and one more, I don't see more usage of journalctl with grep, I think we can fix it locally

Reply:
@NitzanMordhai I have fixed the indent.

Reply:
@shraddhaag lets change the $TIMEOUT to something like $RETRY, it will be easier to understand its retry and not delay

Reply:
Ah yes, that makes sense, my bad. Have changed it to `RETRY_DURATION`.